### PR TITLE
Update to React Native 0.57

### PR DESCRIPTION
**Pull request by rajivshah3**
# Description
Updates to [React Native 0.57](https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#057)

Known issues: 

- The packager cache may need to be cleared (`yarn start --reset-cache`) to resolve errors. See https://github.com/tleunen/babel-plugin-module-resolver/issues/292, https://github.com/tleunen/babel-plugin-module-resolver/issues/310, and https://github.com/facebook/metro/issues/286
- Images and other non-JS resources may not appear in Debug mode. See https://github.com/facebook/metro/issues/289

## Type of change

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Tested on iPhone 6S Simulator
- [x] Tested on physical iPhone 6S
- [x] Tested on iPhone XS Simulator
- [x] Tested on Google Pixel 2 Simulator

# Checklist:

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code
- [x] New and existing unit tests pass locally with my changes
- [x] For changes to `mobile` that include native code (including React Native modules): I have verified that both iOS and Android successfully build in both `Debug` and `Release` modes
- [x] For changes to `shared`: If applicable, I have verified that my changes are implemented correctly in `desktop` and `mobile`

### CODE REVIEW

File: src/mobile/android/gradle.properties [hunk 1]
```diff
@@ -10,7 +10,6 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
-org.gradle.jvmargs=-Xmx1536m
```
Comment (rajivshah3) [inline src/mobile/android/gradle.properties:13]: This shouldn't be deleted

**Comment by rajivshah3** [comment]
~~Blocked by https://github.com/facebook/react-native/issues/21937~~

~~Fixed by using forked version: https://github.com/rajivshah3/react-native/commit/5f3aa84c73389c69cd033c0b4fe69d2a5bbd46cf~~

Now using React Native 0.57.5 🎉

File: src/mobile/android/gradlew.bat [hunk 2]
```diff
@@ -46,9 +46,10 @@ echo location of your Java installation.
 goto fail
 
 :init
-@rem Get command-line arguments, handling Windows variants
+@rem Get command-line arguments, handling Windowz variants
```
Comment (cvarley100) [inline src/mobile/android/gradlew.bat]: Windowz?

**Review by cvarley100 — APPROVED** [review]

File: src/mobile/android/gradlew.bat (hunk 2 above)
Comment (rajivshah3) [reply src/mobile/android/gradlew.bat]: Not sure how that happened 😂 will fix

**Pull request merged**